scripts/pylint_extensions.py

```diff
@@ -422,9 +411,7 @@ def visit_raise(self, node):
                 func_node = property_
 
         doc = docstrings_checker.docstringify(func_node.doc)
-        if not doc.is_valid():
```

Comment:
self.config.accept_no_raise_doc == False and doc.is_valid == False is not possible. Also the other properties(i.e visit_return, visit_yield etc) use similar type of constructs as defined below(line 414)

Reply:
Can you please explain this once again? Why is it not possible?

Reply:
Sorry I had to manually set the value of config.accept_no_raise_doc to false. Reverted this change and added tests!